base/global/thanos-query.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    kubernetes.io/ingress.class: traefik-private
+  name: global-thanos-query
+  namespace: sys-mon
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: global-thanos-query
+  name: global-thanos-query
+  namespace: sys-mon
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "10902"
+    prometheus.io/scrape: "true"
+spec:
+  ports:
+  - port: 9090
+    protocol: TCP
+    targetPort: http
+    name: http
+  selector:
+    app: global-thanos-query
+  sessionAffinity: None
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: global-thanos-query
+  namespace: sys-mon
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: global-thanos-query
+  namespace: sys-mon
```

Comment:
namespace, etc.

What I meant was they should not be in this repository at all, namespace should only be specified at the level of building where an actual namespaces is present, in our case its `kubernetes-manifests`.